test/test_embeddings.py

```diff
@@ -146,7 +151,7 @@ def _check_results(self, embed):
     def test_model_update(self):
         for covariance_type, sigma_shape1 in [
                 ('spherical', 1), ('diagonal', 2)]:
-            embed = sample_embed(covariance_type=covariance_type)
+            embed = sample_embed(energy_type='IP', covariance_type=covariance_type)
```

Comment:
Why was it necessary to change this? I don't see any changes to the update function.